com.unity.render-pipelines.high-definition/Runtime/PostProcessing/PostProcessSystem.cs

```diff
@@ -2268,14 +2269,19 @@ RTHandle Allocator(string id, int frameIndex, RTHandleSystem rtHandleSystem)
             previous = camera.GetPreviousFrameRT((int)HDCameraFrameHistoryType.DepthOfFieldCoC);
         }
 
-        void ReprojectCoCHistory(CommandBuffer cmd, HDCamera camera, bool useMips, ref RTHandle fullresCoC)
+        void ReprojectCoCHistory(CommandBuffer cmd, HDCamera camera, bool useMips, bool maxBlending, ref RTHandle fullresCoC)
         {
             GrabCoCHistory(camera, out var prevCoCTex, out var nextCoCTex, useMips);
             var cocHistoryScale = new Vector2(camera.historyRTHandleProperties.rtHandleScale.z, camera.historyRTHandleProperties.rtHandleScale.w);
 
             //Note: this reprojection creates some ghosting, we should replace it with something based on the new TAA
             ComputeShader cs = m_Resources.shaders.depthOfFieldCoCReprojectCS;
             int kernel = cs.FindKernel("KMain");
+            cs.shaderKeywords = null;
```

Comment:
Looks like you have an old version of staging, the compute shader/kernel stuff have been moved to a PrepareFunction during the move of post process to rendergraph.

Reply:
Indeed, I will rebase my branch and merge any conflicts.

Reply:
You probably know this already, but Github is not updating the code snippets automatically after a rebase, so you have to check the changed files.